src/patterns/ec2-app/base.test.ts

```diff
@@ -389,7 +389,9 @@ describe("the GuEC2App pattern", function () {
           monitoringConfiguration: { noMonitoring: true },
           userData: "",
         })
-    ).toThrowError();
```

Comment:
Should always test the message being thrown, else something else could have been throwing!